pkgs/development/node-packages/default.nix

```diff
@@ -160,6 +160,14 @@ let
 
     node2nix = super.node2nix.override {
       buildInputs = [ pkgs.makeWrapper ];
+      # We need to apply a patch to the source, but buildNodePackage doesn't allow patches.
+      # So we pin the patched commit instead. The commit actually contains two other newer commits
+      # since the last (1.9.0) release, but actually this is a good thing since one of them is a
+      # Hydra-specific fix.
+      src = builtins.fetchTarball {
```

Comment:
```suggestion
      src = fetchTarball {
```

Reply:
Why? Is this always in scope or should I take it from lib?

Reply:
IIRC builtins.fetchTarbal shouldn't be used in nixpkgs.